packages/livechat/src/widget.ts

```
@@ -346,6 +346,15 @@ function setParentUrl(url: string) {
 	callHook('setParentUrl', url);
 }
 
+function setHiddenSystemMessages(hidden: StoreState['iframe']['hiddenSystemMessages']) {
+	if (!Array.isArray(hidden)) {
+		console.log('Error: Invalid parameters. Value must be an array of strings');
```

Wouldn't it be better to throw an error instead?